sample_size/variables.py

```diff
@@ -0,0 +1,52 @@
+class Boolean:
```

Comment:
follow the other comment, how about putting `mde` here as a variable then we can directly using these classes without PowerAnalysisParameters?

Reply:
I hesitate to put `mde` in variable classes since it is a parameter for the calculator not variable.

Reply:
oh ok, i was interpreting each variable as a metric though (it doesn't seem that variable will be used for any other purpose outside of as a "metric" in this repo). and each Variable object can only have one MDE

Reply:
Good point. I can rename the variable classes and let them house mde.